tests/testTriggers.go

```diff
@@ -6987,6 +7000,140 @@ func TriggerIopsBwAsyncDR(contexts *[]*scheduler.Context, recordChan *chan *Even
 	updateMetrics(*event)
 }
 
+func TriggerAsyncDRMigrationSchedule(contexts *[]*scheduler.Context, recordChan *chan *EventRecord) {
+	defer endLongevityTest()
+	startLongevityTest(AsyncDRMigrationSchedule)
+	defer ginkgo.GinkgoRecover()
+	log.InfoD("Async DR Migrationschedule triggered at: %v", time.Now())
```

Comment:
It would be good to print the name of the migration schedule as well.

Reply:
We cannot print here, as its just for trigger information.